src/legacy/core_plugins/vis_type_table/public/legacy_imports.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export { npSetup, npStart } from 'ui/new_platform';
+export { SchemaConfig } from 'ui/visualize/loader/pipeline_helpers/build_pipeline';
+export { getFormat } from 'ui/visualize/loader/pipeline_helpers/utilities';
+export { Vis, AggConfig } from 'ui/vis';
+// @ts-ignore
+export { visFactory, VisFactoryProvider } from 'ui/vis/vis_factory';
+export { AggGroupNames, VisOptionsProps } from 'ui/vis/editors/default';
+// @ts-ignore
+export { Schemas } from 'ui/vis/editors/default/schemas';
+// @ts-ignore
+export { legacyResponseHandlerProvider } from 'ui/vis/response_handlers/legacy';
+
+// @ts-ignore
+export { PrivateProvider } from 'ui/private/private';
+// @ts-ignore
+export { PaginateDirectiveProvider } from 'ui/directives/paginate';
```

Comment:
does this have any other consumer than tablevis ? if no, move it inside.

Reply:
please check all exports below this ... i hope some of them are table vis specific

Reply:
@ppisljar yes, `<paginate>` directive is also used in `saved_object_finder.html`. I checked all exports, there is nothing table vis specific, unfortunately.